packages/o-spreadsheet-engine/src/functions/module_lookup.ts

```diff
@@ -1104,3 +1114,207 @@ export const OFFSET = {
+
+// -----------------------------------------------------------------------------
+// DROP
+// -----------------------------------------------------------------------------
+export const DROP = {
+  description: _t(
+    "Excludes a specified number of rows or columns from the start or end of an array."
+  ),
+  args: [
+    arg("array (range)", _t("The array from which to drop rows or columns")),
+    arg(
+      "rows (number)",
+      _t("The number of rows to drop. A negative value drops from the end of the array.")
+    ),
+    arg(
+      "columns (number, optional)",
+      _t("The number of columns to exclude. A negative value drops from the end of the array.")
+    ),
+  ],
+  compute: function (
+    array: Matrix<{ value: string }>,
+    rows: Maybe<FunctionResultObject>,
+    columns: Maybe<FunctionResultObject>
+  ) {
+    const _rows = toNumber(rows, this.locale);
+    const _columns = toNumber(columns, this.locale);
+    let result = array;
+    if (Math.abs(_columns) >= array.length || Math.abs(_rows) >= array[0].length) {
+      return new EvaluationError(
+        _t(
+          "The number of rows or column to exclude must be smaller than the number of elements in the array."
+        )
+      );
+    }
+    if (_columns >= 0) {
+      result = result.slice(_columns);
+    } else {
+      result = result.slice(0, result.length + _columns);
+    }
+    for (let i = 0; i < result.length; i++) {
+      if (_rows >= 0) {
+        result[i] = result[i].slice(_rows);
+      } else {
+        result[i] = result[i].slice(0, result[i].length + _rows);
+      }
+    }
+    return result;
+  },
+  isExported: true,
+} satisfies AddFunctionDescription;
+
+// -----------------------------------------------------------------------------
+// ARRAYTOTEXT
+// -----------------------------------------------------------------------------
+
+export const ARRAYTOTEXT = {
+  description: _t(
+    "returns an array of text values from any specified range. It passes text values unchanged, and converts non-text values to text."
+  ),
+  args: [
+    arg("array (range)", _t("The array to convert into text")),
+    arg("format (number, default=0)", _t("The format of the returned data."), FORMAT_OPTIONS),
+  ],
+  compute: function (
+    array: Matrix<{ value: string }>,
+    format: Maybe<FunctionResultObject> = { value: 0 }
+  ) {
+    let result = "";
+    const symb = { 0: ",", 1: ";" };
+    const _format = toNumber(format, this.locale);
+    if (!(_format in Object.keys(symb))) {
+      return new EvaluationError(_t("Format must be 0 or 1"));
+    }
+    for (let row = 0; row < array[0].length; row++) {
+      for (let col = 0; col < array.length; col++) {
+        const value = array[col][row].value;
+        let text: string;
+        if (isEvaluationError(value)) {
+          text = value;
+        } else if (typeof value === "string") {
+          text = _format ? `"${value}"` : value;
+        } else {
+          text = toString(value);
```

Comment:
This exact logic is already implemented in `evaluationResultToDisplayString`.
You can extract the code into a helper function and re-use it here